app/views/student_requests/new.html.haml

```diff
@@ -1,43 +1,68 @@
 %h1 Making New Request
-
-= form_tag student_requests_path do
-
-  = label :student_request, :uin , 'UIN'
-  = text_field :student_request, :uin
-
-  = label :student_request, :full_name, 'Full Name'
-  = text_field :student_request, 'full_name'
+.form_new_request
+  = form_tag student_requests_path do
+    %table{:border => "0"}
+      %tr
+        %td
+          = label :student_request, :uin , 'UIN'
+        %td
+          = text_field :student_request, :uin, class: 'input-field'
+      %tr
+        %td
+          = label :student_request, :full_name, 'Full Name'
+        %td  
+          = text_field :student_request, 'full_name', class: 'input-field'
+      %tr
+        %td  
+          = label :student_request, :major, 'Major'
+        %td  
+          = text_field :student_request, 'major', class: 'input-field'
+      %tr
+        %td   
+          = label :student_request, :classification, 'Classification'
+        %td  
+          = select :student_request, 'classification', ['G7', 'G8']
```

Comment:
We could have other classification for undergraduate students right? This is right as per previous code. We can merge this now, but we need to fix this later I guess.